From 4ae47ad7725ab7be5ae7f455f4f51bd133b05430 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@debian.org>
Date: Mon, 29 Aug 2016 00:35:29 -0700
Subject: [PATCH 2/4] omap3_pandora: Switch to using "load" command to load the
 autoboot script.

CONFIG_CMD_FS_GENERIC is enabled; use it to load the autoboot script,
rather than first attempting with fatload and falling back to
ext2load.

Series-to: u-boot
Cover-letter:
Fixes and simplification to omap3_pandora boot configuration

First patch fixes the syntax used for loading a boot script from mmc.

The second patch makes use of the "load" command provided by
CONFIG_CMD_FS_GENERIC.
END
---
 include/configs/omap3_pandora.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/configs/omap3_pandora.h b/include/configs/omap3_pandora.h
index 6ff9375..0d733bd 100644
--- a/include/configs/omap3_pandora.h
+++ b/include/configs/omap3_pandora.h
@@ -93,8 +93,7 @@
 	"mtdparts=" MTDPARTS_DEFAULT "\0" \
 
 #define CONFIG_BOOTCOMMAND \
-	"if mmc rescan && fatload mmc 0:1 ${loadaddr} autoboot.scr || " \
-			"ext2load mmc 0:1 ${loadaddr} autoboot.scr; then " \
+	"if mmc rescan && load mmc 0:1 ${loadaddr} autoboot.scr; then " \
 		"source ${loadaddr}; " \
 	"fi; " \
 	"ubi part boot && ubifsmount ubi:boot && " \
-- 
2.9.3

